wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/Utils.java

```diff
@@ -8,10 +8,11 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.api.workspace.shared;
```

Comment:
It is supposed to be used on GWT side also. Can you elaborate on why do you think it should be moved on servers side?